### DISCUSSION TITLE
lockOutObservationWindow  Attribute

### DISCUSSION THREAD

**Discussion by DevOpserzhao** [go-ldap/ldap#493] [Q&A]
Can I set the window ad Attribute for  lockOutObservationWindow?

**Comment by cpuschma** [comment]
The `msDS-LockoutObservationWindow` is only available for containers with objectClass `[ms-DS-Password-Settings](https://learn.microsoft.com/en-us/windows/win32/adschema/c-msds-passwordsettings)`, known as Password Settings Object (PSO). Once created they're located in `CN=xxxxxxx,CN=Password Settings Container,CN=System,dc=intra,dc=contoso,dc=com `:
```
dn:CN=PSO_AdminUser,CN=Password Settings Container,CN=System,dc=intra,dc=contoso,dc=com
changetype:add
objectClass:msDS-PasswordSettings
msDS-MaximumPasswordAge:-25920000000000
msDS-MinimumPasswordAge:-864000000000
msDS-MinimumPasswordLength:10
msDS-PasswordHistoryLength:24
msDS-PasswordComplexityEnabled:TRUE
msDS-PasswordReversibleEncryptionEnabled:FALSE
msDS-LockoutObservationWindow:-864000000000
msDS-LockoutDuration:0
msDS-LockoutThreshold:10
msDS-PasswordSettingsPrecedence:30
msDS-PSOAppliesTo: CN=DL_ADDS_AdminPWpolicy,ou=groups,ou=_shared,ou=_justice,dc=intra,dc=contoso,dc=com
```
See also: https://blog.joeware.net/2010/03/27/2014/
